src/lib/keypom.ts

```diff
@@ -251,12 +282,24 @@ class KeypomJS {
     pageSize: number,
     getDropErrorCallback?: () => void,
   ) => {
-    let drop: ProtocolReturnedDrop;
     try {
-      drop = await this.getDropInfo({ dropId });
+      /** Get PaginatedDrops caching logic */
+      if (
+        !Object.prototype.hasOwnProperty.call(this.dropStore.paginatedDrops, dropId) ||
+        Date.now() > this.dropStore.paginatedDrops[dropId].getExpiryTime ||
+        pageIndex !== this.dropStore.paginatedDrops[dropId].getPageIndex
+      ) {
+        const newGetPaginatedDropsExpiryTime = new Date(Date.now() + CACHE_MAX_AGE).getTime();
+
+        this.dropStore.paginatedDrops[dropId] = {
+          ...(await this.getDropInfo({ dropId })),
+          getPageIndex: pageIndex,
+          getExpiryTime: newGetPaginatedDropsExpiryTime,
```

Comment:
sorry my bad, can we remove `get` from the variable names. `getXX` sounds like a function name